wolox-training/tag-rails
# Rubocop setup + solved offenses for app directory

## Summary
![](https://github.trello.services/images/mini-trello-icon.png) [Configurar Rubocop](https://trello.com/c/rpVjzvWv/8-configurar-rubocop)

Installed and set up the Rubocop gem for file linting. Once configured the offenses that were found in the `app` directory were resolved.

## Notes
Some modifications were made in regards to the instructions stated on the Trello card due to some functionality in the Rubocop gem being removed in a recent update.
The functionality was moved on to it's own new separate gem `rubocop-rails` which needed to be installed and configured. This changed the command to be executed which can be found in the **Test plan** below.

## Screenshots
![rubocop-ok-screenshot](https://user-images.githubusercontent.com/14864633/61661449-7474c580-aca2-11e9-979c-58e22ff177cb.png)

## Test Plan
* Pull these changes to your local environment
* Run the command `bundle install` to install `rubocop` and `rubocop-rails` gems
* Run the command `bundle exec rubocop --require rubocop-rails app` to check for offenses in the `app` directory
